gui/drawable/inc/drawable.hpp

```diff
@@ -29,17 +27,18 @@ class Drawable {
     const std::unordered_map<std::shared_ptr<Drawable>, Drawable::Position> &getChildDrawables()
```

Comment:
shoudl be const

Reply:
it is const, but clang-format moved "const" word to the next line